sbk-api/src/main/java/io/sbk/api/impl/SbkCSVLogger.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) KMG. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.sbk.api.impl;
+
+import java.io.PrintWriter;
+import java.io.FileWriter;
+
+/**
+ * Class for recoding/printing results to a CSV file called `out.csv`.
+ */
+public class SbkCSVLogger extends SystemLogger {
+    public String fileName = "out.csv";
+
```

Comment:
The out.csv is the default file name; what if user wants to change it another name ? so, override the addargs and parse args .. in addargs you can add option -csvfile option and parseargs you can parse the same

Reply:
Added option `-csvfile` to pass file name. Default is still `out.csv`.